vllm/utils.py

```diff
@@ -232,8 +232,14 @@ def get_open_port() -> int:
             return s.getsockname()[1]
 
 
-def set_cuda_visible_devices(device_ids: List[int]) -> None:
-    os.environ["CUDA_VISIBLE_DEVICES"] = ",".join(map(str, device_ids))
+def update_environment_variables(envs: Dict[str, str]):
+    for k, v in envs.items():
+        if k in os.environ:
+            warnings.warn(
+                f"Overwriting environment variable {k} "
+                f"from {os.environ[k]} to {v}",
+                stacklevel=2)
```

Comment:
Use logger?

Reply:
fixed in [1e62614](https://github.com/vllm-project/vllm/pull/4024/commits/1e62614bd4373cba1ea78f76b9ee47f7ae1d937a)